EssentialFeed/EssentialFeed/Comments/Comment API/RemoteCommentLoader.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  RemoteCommentLoader.swift
+//  EssentialFeed
+//
+//  Created by Robert Dates on 1/21/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+
+public class RemoteCommentLoader: CommentLoader {
```

Comment:
Remove extra line break

```suggestion
import Foundation

public class RemoteCommentLoader: CommentLoader {
```

Reply:
done